Bottom: 1e8608bc091fe53232757329ee1e60011988a412
Top:    e3c9f38f8cf8c33d73e094429f660245e7cc8d8e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-08 11:13:12 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 03bca01f..87e2e447 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -25,8 +25,6 @@
     mariadb_remote_root_access: false
   - role: bacula-director
   - role: bacula-storage
-    tags:
-      - bacula_storage
   - role: bacula-client
   - role: bacula-console
   - role: bacula-script
diff --git a/ansible-runner/Dockerfile b/ansible-runner/Dockerfile
deleted file mode 100644
index 288b5ff2..00000000
--- a/ansible-runner/Dockerfile
+++ /dev/null
@@ -1,40 +0,0 @@
-FROM ubuntu:latest
-
-RUN apt-get update \
-&&  DEBIAN_FRONTEND=noninteractive apt-get install \
-    -y \
-    curl \
-    git \
-    openssh-client \
-    python \
-    python-svn \
-    rsync \
-    sshpass \
-    subversion \
-    vim
-    
-# Install pip
-RUN curl -O https://bootstrap.pypa.io/get-pip.py \
-&&  python get-pip.py --no-cache-dir
-
-RUN pip --no-cache-dir install \
-    'ansible'
-
-ARG UID=1000
-ARG GID=1000
-
-RUN groupadd \
-    --gid ${GID} \
-    --system \
-    ansible \
-&&  useradd \
-    --comment "Ansible User" \
-    --create-home \
-    --gid ${GID} \
-    --uid ${UID} \
-    --system \
-    ansible
-
-WORKDIR /ansible
-VOLUME /ansible
-USER ansible
diff --git a/ansible-runner/build.sh b/ansible-runner/build.sh
deleted file mode 100755
index 96e80c66..00000000
--- a/ansible-runner/build.sh
+++ /dev/null
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-PWD=$(readlink -f $(dirname $0))
-CONTEXT=$(readlink -f "$PWD/..")
-cd $PWD
-
-docker build \
---build-arg UID="$(id -u)" \
---build-arg GID="$(id -g)" \
---pull \
---tag "daedalus-provisioner:latest" \
-"$PWD"
diff --git a/ansible-runner/run.sh b/ansible-runner/run.sh
deleted file mode 100755
index f80680fe..00000000
--- a/ansible-runner/run.sh
+++ /dev/null
@@ -1,19 +0,0 @@
-#!/bin/bash
-
-PWD=$(readlink -f $(dirname $0))
-CONTEXT=$(readlink -f "$PWD/..")
-cd $PWD
-
-exec docker run \
---env ANSIBLE_ROLES_PATH=/ansible \
---init \
---interactive \
---mount type=bind,src="$CONTEXT",dst="/ansible" \
---mount type=bind,src="$HOME/.ssh",dst="/home/ansible/.ssh" \
---network=host \
---rm \
---tty \
---user ansible \
---workdir "/ansible/ansible-playbook" \
-daedalus-provisioner \
-"$@"
diff --git a/provision.sh b/provision.sh
index ad58dd05..3762e89d 100755
--- a/provision.sh
+++ b/provision.sh
@@ -9,6 +9,6 @@ ANSIBLE_SSH_ARGS='-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o Contr
 exec ansible-playbook \
         --connection=ssh \
         --timeout=30 \
-        --inventory-file=/home/coopera/Developer/testdaedalus/vmdaedalus/vagrant_ansible_inventory \
+        --inventory-file=ansible-playbook/staging \
         -vv \
         ansible-playbook/site.yml "$@"
diff --git a/roles/bacula-catalog/tasks/main.yml b/roles/bacula-catalog/tasks/main.yml
index 3deb3fad..2d22328a 100644
--- a/roles/bacula-catalog/tasks/main.yml
+++ b/roles/bacula-catalog/tasks/main.yml
@@ -4,4 +4,3 @@
   register: filecount
 - include_tasks: restore-catalog-with-bsr.yml
   when: filecount.stdout_lines[1] == "0"
-  tags: restore
diff --git a/roles/bacula-restores/README.md b/roles/bacula-restores/README.md
deleted file mode 100644
index 225dd44b..00000000
--- a/roles/bacula-restores/README.md
+++ /dev/null
@@ -1,38 +0,0 @@
-Role Name
-=========
-
-A brief description of the role goes here.
-
-Requirements
-------------
-
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
-
-Role Variables
---------------
-
-A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
-
-Dependencies
-------------
-
-A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
-
-Example Playbook
-----------------
-
-Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
-
-    - hosts: servers
-      roles:
-         - { role: username.rolename, x: 42 }
-
-License
--------
-
-BSD
-
-Author Information
-------------------
-
-An optional section for the role authors to include contact information, or a website (HTML is not allowed).
diff --git a/roles/bacula-restores/defaults/main.yml b/roles/bacula-restores/defaults/main.yml
deleted file mode 100644
index 540bc127..00000000
--- a/roles/bacula-restores/defaults/main.yml
+++ /dev/null
@@ -1,2 +0,0 @@
----
-# defaults file for bacula-restores
\ No newline at end of file
diff --git a/roles/bacula-restores/handlers/main.yml b/roles/bacula-restores/handlers/main.yml
deleted file mode 100644
index 66e6f986..00000000
--- a/roles/bacula-restores/handlers/main.yml
+++ /dev/null
@@ -1,2 +0,0 @@
----
-# handlers file for bacula-restores
\ No newline at end of file
diff --git a/roles/bacula-restores/meta/main.yml b/roles/bacula-restores/meta/main.yml
deleted file mode 100644
index 72237997..00000000
--- a/roles/bacula-restores/meta/main.yml
+++ /dev/null
@@ -1,57 +0,0 @@
-galaxy_info:
-  author: your name
-  description: your description
-  company: your company (optional)
-
-  # If the issue tracker for your role is not on github, uncomment the
-  # next line and provide a value
-  # issue_tracker_url: http://example.com/issue/tracker
-
-  # Some suggested licenses:
-  # - BSD (default)
-  # - MIT
-  # - GPLv2
-  # - GPLv3
-  # - Apache
-  # - CC-BY
-  license: license (GPLv2, CC-BY, etc)
-
-  min_ansible_version: 1.2
-
-  # If this a Container Enabled role, provide the minimum Ansible Container version.
-  # min_ansible_container_version:
-
-  # Optionally specify the branch Galaxy will use when accessing the GitHub
-  # repo for this role. During role install, if no tags are available,
-  # Galaxy will use this branch. During import Galaxy will access files on
-  # this branch. If Travis integration is configured, only notifications for this
-  # branch will be accepted. Otherwise, in all cases, the repo's default branch
-  # (usually master) will be used.
-  #github_branch:
-
-  #
-  # platforms is a list of platforms, and each platform has a name and a list of versions.
-  #
-  # platforms:
-  # - name: Fedora
-  #   versions:
-  #   - all
-  #   - 25
-  # - name: SomePlatform
-  #   versions:
-  #   - all
-  #   - 1.0
-  #   - 7
-  #   - 99.99
-
-  galaxy_tags: []
-    # List tags for your role here, one per line. A tag is a keyword that describes
-    # and categorizes the role. Users find roles by searching for tags. Be sure to
-    # remove the '[]' above, if you add tags to this list.
-    #
-    # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
-    #       Maximum 20 tags per role.
-
-dependencies: []
-  # List your role dependencies here, one per line. Be sure to remove the '[]' above,
-  # if you add dependencies to this list.
\ No newline at end of file
diff --git a/roles/bacula-restores/tests/inventory b/roles/bacula-restores/tests/inventory
deleted file mode 100644
index 878877b0..00000000
--- a/roles/bacula-restores/tests/inventory
+++ /dev/null
@@ -1,2 +0,0 @@
-localhost
-
diff --git a/roles/bacula-restores/tests/test.yml b/roles/bacula-restores/tests/test.yml
deleted file mode 100644
index 3b1d784a..00000000
--- a/roles/bacula-restores/tests/test.yml
+++ /dev/null
@@ -1,5 +0,0 @@
----
-- hosts: localhost
-  remote_user: root
-  roles:
-    - bacula-restores
\ No newline at end of file
diff --git a/roles/bacula-restores/vars/main.yml b/roles/bacula-restores/vars/main.yml
deleted file mode 100644
index 393c5674..00000000
--- a/roles/bacula-restores/vars/main.yml
+++ /dev/null
@@ -1,2 +0,0 @@
----
-# vars file for bacula-restores
\ No newline at end of file
diff --git a/roles/docker-buildsystem/tasks/main.yml b/roles/docker-buildsystem/tasks/main.yml
index 2b626b35..50b4e81c 100644
--- a/roles/docker-buildsystem/tasks/main.yml
+++ b/roles/docker-buildsystem/tasks/main.yml
@@ -120,4 +120,3 @@
 
 - import_tasks: restore.yml
   when: st_buildsystem_restore.stat.exists == False
-  tags: restore
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index 4594a26a..c081706c 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -123,7 +123,6 @@
 
 - include_tasks: restore.yml
   when: st_exim4_restore.stat.exists == False
-  tags: restore
 
 # *****************************************************************************
 # Email accounts
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 38e207ea..92605a06 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -189,4 +189,3 @@
 
 - include_tasks: restore.yml
   when: st_mantisbt_restore.stat.exists == False
-  tags: restore
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 5303effc..96348fdc 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -155,4 +155,3 @@
 
 - include_tasks: restore.yml
   when: st_mediawiki_restore.stat.exists == False
-  tags: restore
diff --git a/roles/docker-openssl/tasks/main.yml b/roles/docker-openssl/tasks/main.yml
index 8f0eaf8c..bf716ae4 100644
--- a/roles/docker-openssl/tasks/main.yml
+++ b/roles/docker-openssl/tasks/main.yml
@@ -97,4 +97,3 @@
 
 - include_tasks: restore.yml
   when: st_openssl_restore.stat.exists == False
-  tags: restore
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index 9b7e0bbd..61e42e4e 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -98,4 +98,3 @@
 
 - include_tasks: restore.yml
   when: st_svn_restore.stat.exists == False
-  tags: restore
